netd.yaml

```diff
@@ -77,10 +77,10 @@ data:
               "ranges": [
                 [
                   {"subnet": podCidr}
-                ]
+                ]ipv6RangeOptional
```

Comment:
Cloud we use some special character($#@ etc.) as the prefix, so that we can easily search/recognize the VARs?
Same for the podCidr.

Reply:
Good idea. I used @, since both # and $ look too much like shell characters.